aws-sns-lambda/module_test.go

```diff
@@ -0,0 +1,15 @@
+package test
+
+import (
+	"testing"
+)
+
+func TestAWSSNSLambda(t *testing.T) {
+	// TODO needs variables set to pass
```

Comment:
We should write an actual test, even if there are no validations. I know that you don't have permissions to run the tests locally, but I will fix that.

Reply:
yeah i didn't have the permissions, so wrote a simple test instead